### DISCUSSION TITLE
请问能否实现通过隧道映射远程内网服务器的端口到本地？

### DISCUSSION THREAD

**Discussion by afateg** [electerm/electerm#3025] [Q&A] [answered]
例如
A：本地电脑，B：可直接访问的公网服务器，C：无法直接访问的内网服务器
我本地A电脑，可以通过SSH登录到B服务器上，想要通过SSH隧道，将C服务器的端口映射到A本地端口上，请问使用最新版的electerm能否实现？

直接的SSH命令是：
ssh -L A_local_port:C_server_ip:C_server_port B_user@B_server_ip

**Comment by zxdong262** [comment]
可以的，创建书签时候设置ssh隧道和连接跳跃即可，跳跃到最终服务器后就会建立端口映射，如下图的例子即可通过访问localhost:1333实现访问192.168.2.27:1444
![Screen Shot 2023-10-09 at 11 49 35](https://github.com/electerm/electerm/assets/1641949/1d3d83c4-75af-451b-9649-fcb471df5b40)
![Screen Shot 2023-10-09 at 11 49 50](https://github.com/electerm/electerm/assets/1641949/da12c608-4b92-4531-a66c-654c937a4fd6)

**Reply by afateg** [reply, marked as answer]
十分感谢！
